tests/queries/0_stateless/00926_geo_to_h3.sql

```diff
@@ -10,11 +10,11 @@ INSERT INTO table1 VALUES(55.72076200, 37.59813500, 15);
 INSERT INTO table1 VALUES(55.72076201, 37.59813500, 15);
```

Comment:
I believe that there would be more places where `geoToH3` is used where this change might be necessary:
```
 grep -l "geoToH3(" -r ./queries/0_stateless
./queries/0_stateless/00926_geo_to_h3.sql
./queries/0_stateless/03404_geotoh3_input_order.sql
./queries/0_stateless/01906_h3_to_geo.sql
./queries/0_stateless/02021_h3_is_res_classIII.sql
./queries/0_stateless/02160_h3_cell_area_m2.sql
./queries/0_stateless/02160_h3_cell_area_rads2.sql
./queries/0_stateless/02223_h3_test_const_columns.sql
./queries/0_stateless/02509_h3_arguments.sql
```
The stateless tests should point out where the other changes might be needed.

Reply:
https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=78852&sha=9902d375f356b0b1b4a7c1ac20f2f24b61b9ee30&name_0=PR&name_1=Stateless+tests+%28release%29